common/network-shuffle/src/test/java/org/apache/spark/network/shuffle/ExternalShuffleIntegrationSuite.java

```diff
@@ -186,7 +186,12 @@ public void testFetchThreeSort() throws Exception {
 
   @Test
   public void testFetchInvalidShuffle() throws Exception {
-    registerExecutor("exec-1", dataContext0.createExecutorInfo("unknown sort manager"));
+    try {
```

Comment:
I think this test the way it's written doesn't make much sense any more since registerExecutor throws an exception. Personally, I'd prefer if we stopped checked for blocks being empty, and rather assert that the correct type of exception with the correct message is being thrown.